README.md

```diff
@@ -4,6 +4,6 @@
 
 # chproxy
 
-Chproxy is an http proxy and load balancer for [ClickHouse](https://ClickHouse.yandex) database. 
+Chproxy is an HTTP proxy and load balancer for the [ClickHouse](https://ClickHouse.yandex) database.
```

Comment:
```suggestion
Chproxy is an HTTP proxy and load balancer for [ClickHouse](https://ClickHouse.yandex).
```
Actually this suggestion is more consistent with https://github.com/ContentSquare/chproxy/pull/145/files#diff-4887c6195f633baa80b7eefd1b33ca781a4621ce9e9e6b45c9ae99153f1dc5b4R9